packages/f/filc/xmake.lua

```diff
@@ -0,0 +1,47 @@
+package("filc")
+    set_kind("toolchain")
+    set_homepage("https://fil-c.org/")
+    set_description("A memory safe implementation of the C and C++ programming languages.")
+    set_license("LLVM")
+
+    if is_host("linux") then
+        if os.arch() == "x86_64" then
+            set_urls("https://github.com/pizlonator/fil-c/releases/download/v$(version)/filc-$(version)-linux-x86_64.tar.xz")
+            add_versions("0.674", "a8ec349f383a49dacc09a9540643164c67081245e591c1e6823609653f2c9740")
+        end
+    end
+
+    add_deps("patchelf") -- needed for setup.sh
+
+    on_install("@linux|x86_64", function (package)
+        local installdir = package:installdir()
+        os.cp(path.join(os.curdir(), "*"), installdir)
+
+        -- now we replicate the commands in setup.sh...
+        local pathx = path.join(installdir, "/pizfix/lib64") .. ":" .. path.join(installdir, "/pizfix/lib")
+        os.runv("patchelf --set-rpath", {pathx, "pizfix/lib/libc.so"}, {shell=true})
```

Comment:
patchelf is not shell script file, we need not to set `{shell = true}`

Reply:
```lua
os.runv("patchelf", {"--set-rpath", pathx, "pizfix/lib/libc.so"})
```

Reply:
Thanks, will do.